prometheus-exporters/openstack-exporter/values.yaml

```diff
@@ -1,4 +1,6 @@
 owner-info:
+  support-group: compute-storage-api
+  service: cinder
```

Comment:
The [owner-info template](https://github.com/sapcc/helm-charts/blob/cinder_update_alerts/common/owner-info/templates/configmap.yaml#L7) expects a string and not a collection. This commit fixes the problem.